.github/workflows/ci-cd-automation.yml

```
@@ -0,0 +1,27 @@
+name: Servicefoundry CI/CD Automation Example
+
+on:
+  push:
+    paths:
+      - 'ci-cd-automation/*'
+
+jobs:
+  production_deploy:
+    runs-on: ubuntu-latest
+    env:
+      TFY_HOST: https://app.develop.truefoundry.tech
+      TFY_API_KEY: ${{ secrets.TFY_API_KEY }}
+      WORKSPACE_FQN: tfy-dev-cluster:ci-cd-examples
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up python
+        uses: actions/setup-python@v3
+        with:
+          python-version: 3.9
+
+      - name: Install servicefoundry-cli library
+        run: pip install servicefoundry
```

pin to a specific version range
```suggestion
        run: pip install "servicefoundry>=0.2.14,<0.3.0"
```